cmd/codebuild/reset/service_test.go

```diff
@@ -56,7 +56,9 @@ func TestService(t *testing.T) {
 		t.Run("should be a singleton", func(t *testing.T) {
 			svc := &service{}
 
-			require.True(t, svc.config() == svc.config())
+			firstRes := svc.config()
```

Comment:
linter didn't like checking against `svc.config() == svc.config()`.
I would disagree with the linter in this case, but it was an easy enough fix